include/alphablend.h

```diff
@@ -1,6 +1,8 @@
 #ifndef _ALPHABLEND_H
 #define _ALPHABLEND_H
+/* This line sets the WINVER to Windows 2000
```

Comment:
just remove this, don't see any reason for it here

Reply:
I have removed it entirely.